README.md

```diff
@@ -1,2 +1,34 @@
 # lab-16
 TCP Server / Message Application
+
+### Links and Resources
+* [submission PR](https://github.com/meron-401n14/lab-16/pull/1)
+* [travis](https://www.travis-ci.com/meron-401n14/lab-16)
+
+#### Documentation
+* [jsdoc](comments only)
+
+### Modules
+* server
+* logger
+* app
+#### Running the app
+* node app.js ./data/data.text
```

Comment:
To start the file-changer it is:   `node file-changer/app.js  <path-to-my-file>` . In your case it is `file-changer/data/data.txt`